packages/ic-management/src/ic-management.canister.ts

```diff
@@ -175,4 +176,28 @@ export class ICManagementCanister {
    */
   deleteCanister = (canisterId: Principal): Promise<void> =>
     this.service.delete_canister({ canister_id: canisterId });
+
+  /**
+   * Creates a canister. Only available on development instances.
```

Comment:
dumb question: what's "development instances"? testnet?

Reply:
I'm not sure about testnets to be honest, but the DFX replica is definitely a development instance. That's where I'm using this.

Reply:
Ah in that sense, gotcha!